submissions/kolawoleOgunfowokan/HTML/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please exclude this from the PR. I think someone once added  `.vscode` to gitignore. We generally don't want to check-in this file

Reply:
The .vscode file has been excluded. Thanks